plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/ConclaveTask.kt

```diff
@@ -94,6 +94,15 @@ abstract class ConclaveTask : DefaultTask() {
         }
     }
 
+    fun commandWithResultAndOutput(
```

Comment:
I'm actually not sure we need these command helpers. The Gradle API has the `project.exec` feature. If you need the exec result then perhaps just use that directly rather than creating another method.